packs/salt/tests/test_action_local.py

```diff
@@ -0,0 +1,64 @@
+from st2tests.base import BaseActionTestCase
+
+from local import SaltLocal
+import requests_mock
+from requests_mock.contrib import fixture
+import testtools
+
+__all__ = [
+    'SaltLocalActionTestCase'
+]
+
+no_args = {
+    'module': 'this.something',
+    'target': '*',
+    'expr_form': 'glob',
+    'args': []
+}
+
+one_arg = {
+    'module': 'this.something',
+    'target': '*',
+    'expr_form': 'glob',
+    'args': ['os'],
+}
+
+multiple_args = {
+    'module': 'this.something',
+    'target': '*',
+    'expr_form': 'glob',
+    'args': ['this', 'that', 'home'],
+}
+
+CONFIG_DATA = {
+    'api_url': 'https://example.com',
+    'username': 'this',
+    'password': 'that'
+}
+requests_mock.Mocker.TEST_PREFIX = 'test'
+
+
+class SaltLocalActionTestCase(testtools.TestCase, BaseActionTestCase):
```

Comment:
Yeah, I think just setting it to mock should also work, but I'm fine with your current approach as well :)

This way in the future we can also assert on the URL parameters and post data if needed :)